### PR TITLE
Added rest and update CLI commands

### PR DESCRIPTION
**Pull request by jsmartt**
This adds the 'rest' and 'update' commands to the CLI, and removes the `--force' option from the create_from_file command (if you don't want to update it, just pass the 'if_missing' flag).

### CODE REVIEW

**Comment by jsmartt** [comment]
Hmm. Something isn't quite right here. It's showing 611 examples on my machine (and 92% coverage), but only 588 examples (and 90.6% coverage) on Travis.

**Comment by jsmartt** [comment]
@tmiotto , I added a couple of examples to the README. I'm hoping the main way users will get usage information is to just run `$ oneview-sdk-ruby help`. There's more examples in there, and it shows the available options, etc.

**Comment by tmiotto** [comment]
@jsmartt Yep, I know it, but the user may pass through this functionality since all the others are mainly directed to the resources actions, and this one is more specific to the client actions, so it is good to have it explicit there in the README

File: README.md [hunk 1]
```diff
@@ -264,6 +264,20 @@ $ oneview-sdk-ruby create_from_file /my-server-profile.json
...
+```bash
+$ oneview-sdk-ruby update FCNetwork FC1 -h linkStabilityTime:20  # Using hash format
+$ oneview-sdk-ruby update Volume VOL_01 -j '{"shareable": true}' # Using json format
+```
+
+##### Make REST calls:
+
+```bash
+$ oneview-sdk-ruby rest get rest/fc-networks
+$ oneview-sdk-ruby rest PUT rest/enclosures/<id>/configuration
```
Comment (tmiotto) [inline README.md]: We're expecting the users to use the verbs in the case they want?

Comment (jsmartt) [inline README.md]: Ah. I would have made them the same, but I missed that (copy and paste). But yeah, case really doesn't matter, so users can use whatever they prefer

**Pull request merged**